pulumi/python/utility/kic-image-build/ingress_controller_image_base_provider.py

```diff
@@ -47,7 +47,8 @@ def _docker_pull(self, image_name: str) -> str:
         :param image_name: full container image name in the format of repository:tag
         :return full image name with server name (e.g. docker.io/library/debian:buster-slim)
         """
-        cmd = f'docker pull --quiet "{image_name}"'
+
+        cmd = f'docker pull --platform linux/amd64 --quiet "{image_name}"'
```

Comment:
**annotation**:
Necessary to make sure we get the version of the image that will be compatible with the deployment environment not the computer running it.